demo/README.md

```diff
@@ -0,0 +1,33 @@
+# Demo for Pantheon XDG Desktop Portals
+
+A demo application to test pantheon xdg desktop portals.
+
+## Dependencies
+
+You'll need the following dependencies to successfully build the demo application:
+
+    sudo apt install elementary-sdk
+    flatpak install --user --assumeyes appcenter io.elementary.Platform io.elementary.Sdk
+
+## Install
+
+Run `flatpak-builder` to build and install the application:
+
+    flatpak-builder build io.elementary.portals.demo.yml --user --install --force-clean
```

Comment:
The path to the `yml` file needs to be `demo/io.elementary.portals.demo.yml` if you are in the root directory.